kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_faq.adoc

```diff
@@ -121,3 +121,27 @@ spec:
 
 Where the above defines the `PersistentVolumeClaim` called `foo` which is then mounted into the container at `/whatnot`
 
+=== How do I generate Ingress for my generated Service?
+`Ingress` generation is supported by Eclipse JKube for `Service` objects of type `LoadBalancer`. In order to generate `Ingress` you need to enable `jkube.createExternalUrls` property to `true` and `jkube.domain` property to desired host suffix, it would be appended to your service name for host value. You can also also provide a host for it in XML config like this:
+```
+  <plugin>
+    <groupId>org.eclipse.jkube</groupId>
+    <artifactId>kubernetes-maven-plugin</artifactId>
+    <version>${project.version}</version>
```

Comment:
Please use Asciidoc variables that will be replaced upon documentation generation
```suggestion
    <artifactId>{plugin}</artifactId>
    <version>{version}</version>
```